assets/js/modules/adsense/datastore/index.js

```diff
@@ -62,6 +69,17 @@ export const actions = Data.addInitializeAction( Data.collectActions(
 	report.actions,
 	tags.actions,
 	urlchannels.actions,
+	settings.actions,
+	adblocker.actions,
+	{
+		// TODO: Revisit better way to handle and retrieve errors.
+		receiveError( error ) {
+			return {
+				type: 'RECEIVE_ERROR',
+				payload: { error },
+			};
+		},
```

Comment:
True, but this is good to have for testing, we actually need to use it in AdSense in some places because certain account states rely on expected errors. I'd argue we should potentially add this to other stores too (not as part of this PR).